src/getModuleLoader.ts

```diff
@@ -44,12 +45,15 @@ type getModuleLoaderType = <T, R extends AsmRuntimeType>(
  *
  * @param {{[key: string]: any}} [module] Stringmap object to be injected into wasm runtime for override / set additional value in asm module.
  *
+ * @param {{[key: string]: any}} [initializeOpts] Stringmap object to be used to configure the intialization of the module.
+ *
  * @returns {moduleLoaderType<T>} Loader function
  */
 const getModuleLoader: getModuleLoaderType = <T, R extends AsmRuntimeType>(
   factoryLoader: (runtime: R, environment: ENVIRONMENT) => T,
   runtimeModule: runtimeModuleType,
-  module?: StringMap
+  module?: StringMap,
+  initializeOpts?: StringMap
```

Comment:
and we can destruct directrly:

```
(... {timeout}: Partial<ModuleInitOption>)
```

Reply:
You'll have to excuse my absolute naivety on this, my typescript-foo is weak. Not too sure what I'm missing here...

```diff
   module?: StringMap,
-  initializeOpts?: StringMap
+  initOptions: (... {timeout}: Partial<ModuleInitOption>)
 ) => async (environment?: ENVIRONMENT) => {
```
```
src/getModuleLoader.ts(64,1): error TS1005: '=>' expected.
```

Reply:
`...` was just to skip rest of params, so what I meant was

```ts
type getModuleLoaderType = <T, R extends AsmRuntimeType>(
  factoryLoader: (runtime: R, environment: ENVIRONMENT) => T,
  runtimeModule: runtimeModuleType,
  module?: StringMap,
  {timeout}: Partial<InitOptions> = {}
) => moduleLoaderType<T>;
```

Reply:
Thank you :-)